HW0/P4/P4.py

```diff
@@ -30,48 +72,106 @@
     # Load true trajectory and plot it
     # Normally, this data wouldn't be available in the real world
     #####################
+    x_true, y_true, z_true, vx, vy, vz = np.loadtxt("P4_trajectory.txt", delimiter=",", unpack=True)
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '--b', label='True trajectory')
+    ax.plot(x_true, y_true, z_true,
+            '--b', label='True trajectory')
 
     #####################
     # Part 2:
     #
     # Read the observation array and plot it (Part 2)
     #####################
+    x_obs, y_obs, z_obs = np.loadtxt("P4_measurements.txt", delimiter=",", unpack=True)    
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '.g', label='Observed trajectory')
+    ax.plot(x_obs, y_obs, z_obs,
+            '.g', label='Observed trajectory')
 
```

Comment:
I'm super impressed with how clean your code is and how well you've laid it out!